src/twisted/internet/unix.py

```diff
@@ -171,10 +171,22 @@ def doRead(self):
             else:
                 return main.CONNECTION_LOST
 
-        if ancillary:
-            ancillaryData = ancillary[0][2]
-            fdCount = len(ancillaryData) // 4
-            fds = struct.unpack('i'*fdCount, ancillaryData)
+        for cmsgLevel, cmsgType, cmsgData in ancillary:
+            if (cmsgLevel != socket.SOL_SOCKET or
+                cmsgType != sendmsg.SCM_RIGHTS):
+                log.msg(
```

Comment:
(optional)

It seems like it should be the case that new code should use `twisted.logger` instead of `twisted.python.log`.  I don't know if this is really true.  It would probably be a good thing to document somewhere.

That said, this module already uses `twisted.python.log` everywhere so this message fits in with the existing code fairly well.  Unless there's an explicit policy to the contrary, I don't have a problem, with leaving this as-is and doing the logging conversion work at some later point.